packages/components/src/Tooltip/Tooltip.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import { generateSnapshots } from "@chanzuckerberg/story-utils";
+import { userEvent } from "@storybook/testing-library";
+import { composeStories } from "@storybook/testing-react";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import * as TooltipStoryFile from "./Tooltip.stories";
+
+const { Interactive } = composeStories(TooltipStoryFile);
+
+describe("<Tooltip />", () => {
+  generateSnapshots(TooltipStoryFile, {
+    getElement: (wrapper) => {
+      return wrapper.baseElement;
+    },
+  });
+
+  it("should close tooltip via escape key", async () => {
+    // @ts-expect-error Animation prop exists on Tippy but not exposed on Tooltip,
```

Comment:
Would another solution here be to add `animation` to the props list?

Reply:
Oh that would be a good option but: 
1. I didn't want to leave the scope of this ticket 
2. Are there animation guidelines that we can default animations for?

Reply:
I don't think we have any guidelines yet, but [animation type from Tippy](https://github.com/atomiks/tippyjs/blob/8f7b2df156ae39c4df23c481b77ab59fe7f08c44/src/types.ts#L50) is `string | boolean` and I believe the default is `"fade"`. I think if we wanted to use a different one, [it would need to be imported](https://atomiks.github.io/tippyjs/v6/animations/). I don't expect people to really be playing with different animations, but I could see there being other situations where someone needed to remove it.

Reply:
Ah reasonable that animation might want to be disabled. Added the `animation` prop